drivers/ieee802154/Kconfig.nrf5

```diff
@@ -72,7 +72,7 @@ config IEEE802154_NRF5_FCS_IN_LENGTH
 
 config IEEE802154_NRF5_DELAY_TRX_ACC
 	int "Clock accuracy for delayed operations"
-	default CLOCK_CONTROL_NRF_ACCURACY if CLOCK_CONTROL_NRF_ACCURACY < 255
+	default CLOCK_CONTROL_NRF_ACCURACY if (CLOCK_CONTROL_NRF && (CLOCK_CONTROL_NRF_ACCURACY < 255))
```

Comment:
Just note, not problem here. Here is just how 802154 is integrated and deals with missing data.

@nordic-krch  wouldn't it be better that `CLOCK_CONTROL_NRF_ACCURACY` be defined in all cases there: https://github.com/zephyrproject-rtos/zephyr/blob/42c8b7fb00a8f6343966e140b4bba593bb92cfdb/drivers/clock_control/Kconfig.nrf#L162
some fallback value?

Reply:
right, fallback value should be there. but which? I guess that lowest (meaning highest value 500)

Reply:
Is the unit of this value PPM? If so, I agree the safe default should be higher. Maybe even in the range of 1000s

Reply:
@hubertmis please note that for the purpose of ieee802154 subsystem, a larger value of this Kconfig will currently not translate to a representation of worse accuracy. The ieee802154 radio API specifies the clock accuracy as `uint8_t`:
https://github.com/zephyrproject-rtos/zephyr/blob/6a42ede43b53ceffddeaa814b4bd99284c25240d/include/zephyr/net/ieee802154_radio.h#L1856-L1858

I will happily follow this up and address both the comment from @ankuns and from you but I'd rather do it out of scope of this PR. Here the only goal was to handle missing data for nRF54H20 as noted above.

Reply:
Sure, I don't think that's a big issue. Can be refactored later. That's why I approved anyway.